infrastructure/kube/templates/relay-maintainer/initcontainer/provision-relay-maintainer.js

```diff
@@ -0,0 +1,115 @@
+const fs = require('fs')
+const Web3 = require('web3')
+const HDWalletProvider = require("@truffle/hdwallet-provider")
+
+const URL = require('url')
+
+// ETH host info
+const ethRPCUrl = process.env.ETH_RPC_URL
+const ethNetworkId = process.env.ETH_NETWORK_ID
+
+// Relay contract info
+const RelayJSON = require("@keep-network/tbtc/artifacts/Relay.json")
+
+// Contract owner info
+const contractOwnerAddress = process.env.CONTRACT_OWNER_ETH_ACCOUNT_ADDRESS
+const purse = contractOwnerAddress
+
+const contractOwnerPrivateKey = process.env.CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY
+const contractOwnerProvider = new HDWalletProvider(
+  contractOwnerPrivateKey,
+  ethRPCUrl,
+)
+
+const operatorKeyFile = process.env.RELAY_MAINTAINER_ETH_KEYFILE_PATH
+
+/*
+We override transactionConfirmationBlocks and transactionBlockTimeout because they're
+25 and 50 blocks respectively at default.  The result of this on small private testnets
+is long wait times for scripts to execute.
+*/
+const web3_options = {
+  defaultBlock: 'latest',
+  defaultGas: 4712388,
+  transactionBlockTimeout: 25,
+  transactionConfirmationBlocks: 3,
+  transactionPollingTimeout: 480
+}
+
+const web3 = new Web3(contractOwnerProvider, null, web3_options)
+
+async function provisionRelayMaintainer() {
+  console.log('###########  Provisioning relay maintainer! ###########')
+
+  console.log(`\n<<<<<<<<<<<< Read operator address from key file >>>>>>>>>>>>`)
+  const operatorAddress = readAddressFromKeyFile(operatorKeyFile)
```

Comment:
It doesn't look like we need the keyfile on the relay-maintainer..rather we're just grepping data out of the keyfile.

We can read the account address directly from a ConfigMap., so this can go away in favor of an env var.  Will post an example in the Deployment file